src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -942,4 +942,68 @@ public function spin($lambda, $wait = 60) {
             $backtrace[1]['function']
         ));
     }
+
+	/**
+	 * Check the value set in an input textbox(visible)
+	 * @example: Then the value in the "username" field should be "John Smith"
+	 * @Then /^the value in the "(?P<field>(?:[^"]|\\")*)" field should be "(?P<value>(?:[^"]|\\")*)"$/
+	 */
+	public function theValueInFieldShouldBe($field, $value) {
```

Comment:
@chillu, they are different. The existing one is check the value matched partially. My new one is to check the value matched exactly. I will update it later. Thanks!